conbench/templates/benchmark-result.html

```diff
@@ -10,16 +10,21 @@
     <li class="breadcrumb-item active">
       <a href="{{ url_for('app.batch', batch_id=benchmark.batch_id) }}">Batch</a>
     </li>
-    <li class="breadcrumb-item active aria-current="page">Benchmark result {{ benchmark.id }}</li>
+    <li class="breadcrumb-item active aria-current="page">
+      <a href="{{ url_for('app.benchmark', benchmark_id=benchmark.id) }}">
+      Benchmark result {{ benchmark.id[:7] }}...</a>
+    </li>
```

Comment:
This is for #865 CC @jonkeane 
![image](https://user-images.githubusercontent.com/265630/228216797-887a0e32-de0c-4011-8090-407540fb0e53.png)